storm-client/src/jvm/org/apache/storm/utils/Utils.java

```diff
@@ -363,7 +358,7 @@ public void run() {
                 } catch (Throwable t) {
                     if (Utils.exceptionCauseIsInstanceOf(
                             InterruptedException.class, t)) {
-                        LOG.info("Async loop interrupted!");
+                        LOG.error("Async loop interrupted!", t);
```

Comment:
Here we assume that it is not an error, so if my understanding is right, `info` level would make more sense. (I'd second @srdo.)